src/modules/feature-modules/innovator/pages/innovation/export-requests/export-request-reject.component.ts

```diff
@@ -32,16 +36,16 @@ export class InnovationExportRequestRejectComponent extends CoreComponent implem
     this.innovationId = this.activatedRoute.snapshot.params.innovationId;
     this.requestId = this.activatedRoute.snapshot.params.requestId;
 
+    this.setPageTitle('Reject request', { showPage: false });
+    this.setBackLink('Go back', `/innovator/innovations/${this.innovationId}/record/export-requests/${this.requestId}`);
+
   }
 
   ngOnInit(): void {
 
-    this.setPageTitle('Reject request', { size: 'l', width: '2.thirds' });
-    this.setBackLink('Go back', `/innovator/innovations/${this.innovationId}/export/${this.requestId}`);
-
     this.innovationsService.getExportRequestInfo(this.innovationId, this.requestId).subscribe(response => {
 
-      this.request = response;
+      this.innovationRequest = response;
```

Comment:
I think it would be beneficial to double-check if the request is `PENDING`, and if not redirect him back.  If we don't this validation we better catch the error on `onSubmit()` function.